ros-hsrc_description/install.yaml

```diff
@@ -0,0 +1,7 @@
+- type: target
+  name: hsr-setup
+
+- type: ros
+  source:
+    type: system
+    name: hsrc-description
```

Comment:
Why do we need the description of a robot we do not use?

Reply:
Because one of the hsrb_moveit pkgs does depend on it. Naming and dependencies aren't always consistent.